client/verta/tests/test_dataset_versioning/test_dataset.py

```diff
@@ -71,6 +73,17 @@ def test_attributes(self, client, created_entities):
 
 
 class TestCreateGet:
+    def test_creation_updates_dataset_timestamp(self, client, dataset):
+        """Version creation should update its dataset's time_updated field."""
+        time_updated = dataset._msg.time_updated
+
+        dataset_version = dataset.create_version(Path(["conftest.py"]))
+
+        time.sleep(30)  # wait for reconciler
+        dataset._fetch_with_no_cache()
+        assert dataset._msg.time_updated > time_updated
+        assert dataset._msg.time_updated == dataset_version._msg.time_logged
+
```

Comment:
This is the main change—everything else is a cleanup.

Reply:
@conradoverta Is it unreliable to wait like this for the reconciler to run?

Reply:
Potentially. I don't recall what sync period we have today. I do know it's not immediate.

Reply:
I've bumped the wait period from 30sec to 60sec, which seems to work. Can be adjusted in the future as needed.